pkg/resources/monitoring/grafana.go

```diff
@@ -175,9 +175,14 @@ func (r *Reconciler) grafana() ([]resources.Resource, error) {
 		return nil, fmt.Errorf("invalid grafana hostname: %w", err)
 	}
 
+	tag := "latest"
```

Comment:
do we really want to pin this to `latest` ? I'm assuming any major changes to datasource apis will be matched in the latest cortex version, so I'm also assuming this is ok

Reply:
Older versions didn't have the ability to set the tag, so if it's not set in the config the default is `latest` to match the old behavior. It can (and probably should) be pinned to a specific version, but that should be configured elsewhere. `latest` is semantically equivalent to empty tag so it's a reasonable default.